internal/policy/source/repo.go

```diff
@@ -56,6 +56,7 @@ func (p *PolicyRepo) GetPolicyDir() string {
 func (p *PolicyRepo) GetPolicies(ctx context.Context, dest string) error {
 	// Checkout policy repo into work directory.
 	log.Debugf("Checking out repo %s at %s to work dir", p.RepoURL, p.RepoRef)
+	// Todo: Consider if we can use downloader.Download here instead
```

Comment:
I'm interested in the idea of doing all our downloads with conftest downloader, but not sure I'm ready to take on the patch.

Reply:
+1 I think it would be a nice idea to explore.